ingestion/src/metadata/ingestion/source/database/bigquery.py

```diff
@@ -223,7 +223,7 @@ def get_database_names(self) -> Iterable[str]:
         ):
             self.set_inspector(database_name=self.project_ids)
             yield self.project_ids
-        if hasattr(
+        elif hasattr(
```

Comment:
this statement change will now make sure to not send single project ids into else below and ingest only ones